src/main/app/src/app/plan-items/model/plan-item.ts

```diff
@@ -18,4 +18,5 @@ export class PlanItem {
     zaakUuid: string;
     userEventListenerActie: UserEventListenerActie;
     toelichting: string;
+    fataledatum: string;
```

Comment:
in HumanTaskData heet ie fataleDatum. Het zijn ook twee woorden. Dus zou tocj moeten zijn fataleDatum?